src/System.Drawing.Common/src/System/Drawing/Imaging/EncoderParameter.cs

```diff
@@ -447,6 +447,9 @@ public EncoderParameter(Encoder encoder, int numberValues, EncoderParameterValue
                 case EncoderParameterValueType.ValueTypeRationalRange:
                     size = 2 * 2 * 4;
                     break;
+                case EncoderParameterValueType.ValueTypePointer:
```

Comment:
This branch uses `ValueTypePointer`, which was added to `EncoderParameterValueType` (public API) as part of this PR. That value is required to implement the fix. I can hard-coded it to 9, if you want to keep the API & the fix separate, though.

Reply:
Hmm I think we should keep the fix separate from this PR until the API is approved.

Reply:
OK, I reverted that change for now. Any idea how long it would take for that API to get reviewed?

Reply:
No idea. I just marked it as read-for-review, and it is marked as ready-for-review, so it is just a matter of waiting for the API Review team to get to it.